src/Text.Analyzers/Setup/Text.Analyzers.Setup.csproj

```diff
@@ -11,12 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Core\Text.Analyzers.csproj">
-      <Name>Text.Analyzers</Name>
-      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup</IncludeOutputGroupsInVSIX>
-      <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup</IncludeOutputGroupsInVSIXLocalOnly>
-      <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
```

Comment:
Are these metadata properties not needed anymore? The other project references still have them. Have you verified that the resulting VSIXes are correct?

Reply:
I didn't modify this file by hand, something in VS must have changed this.